.github/actions/rancher-interaction/action.yml

```diff
@@ -0,0 +1,52 @@
+name: "rancher-interaction"
+description: "register or detach cluster to rancher"
+inputs:
+  rancher-access-key:
+    description: 'Rancher API access key'
+    required: true
+  rancher-secret-key:
+    description: 'Rancher API secret key'
+    required: true
+  rancher-url:
+    description: 'Rancher URL'
+    required: true
+  option:
```

Comment:
maybe action is more self-explanatory?

Reply:
updated descriptions and added to README file

Reply:
I mean, renaming from option to action may be an improvement

Reply:
on:
  workflow_dispatch:
    inputs:
      name:
        type: choice
        description: Who to greet
        options: 
        - monalisa
        - cschleiden
        
        
 if you mean smth like above, i was trying, but in action is not working like in worflow_dispatch. 
 
 If you mean only change name from option to action i can do that :)